app/services/bulk_republisher.rb

```diff
@@ -41,11 +43,31 @@ def republish_all_documents_with_publicly_visible_editions_with_html_attachments
     republish_by_document_ids(document_ids)
   end
 
+  def republish_all_by_type(type)
+    begin
+      content_type = type.constantize
```

Comment:
1. For increased safety, I guess we could search for the type in `republishable_content_types` from the helper before constantising?
2. Lower down when seeing `content_type.pluck` I was a little confused about plucking on a string. Is it worth calling this `type_klass`, `type_model`, `content_type_klass`, or `content_type_model`?